frontend/src/scenes/onboarding/OnboardingProductIntroduction.tsx

```diff
@@ -192,6 +194,7 @@ export function OnboardingProductIntroduction({ stepKey }: { stepKey: Onboarding
         [ProductKey.EXPERIMENTS]: 'experimentation',
         [ProductKey.PRODUCT_ANALYTICS]: 'product-analytics',
     }
+    const product = billing?.products.find((product) => product.type === productKey)
```

Comment:
I updated, the logic now has a billingProduct we can use for this.

The main change here is that onboarding now works with or without billing, so in a lot of the cases we don't know if we have billing info at all any more. We see it a lot in this page because it does depend on a billing response - but we have to wait for the response to be received before anything can render.